### PR TITLE
209879 update eds packages

### PR DESCRIPTION
**Pull request by Pespiri**

### CODE REVIEW

File: src/components/page-configuration/style.css [hunk 1]
```diff
@@ -1,26 +1,13 @@
...
-@media (min-width: 20rem) {
-  .scrim .delete-app {
-    width: 18rem;
-  }
-}
-
-@media (min-width: 50rem) {
-  .scrim .delete-app {
-    width: 25rem;
+@media (max-width: 30rem) {
```
Comment (oleterno-equinor) [inline src/components/page-configuration/style.css:9]: The smallest screen size should have the initial styling. Media queries should be used to accommodate larger screens using `min-width`

File: src/components/page-configuration/style.css [hunk 2]
```diff
@@ -1,26 +1,13 @@
...
+  .delete-app-content {
+    width: 95vw;
   }
```
Comment (oleterno-equinor) [inline src/components/page-configuration/style.css:12]: Style the accordion panel to not have `min-height: 96px` as it breaks the styling of accordions with only one line of text such as "Private image hubs" and "Build secrets"

File: src/components/page-create-application/style.css [hunk 3]
```diff
@@ -12,33 +12,14 @@
...
   padding: 0 var(--eds_spacing_medium);
-}
-
-.dialog-container .dialog-content {
-  height: auto;
-  max-height: 70vh;
-  max-width: 50rem;
-  padding: var(--eds_spacing_small);
-  overflow-x: auto;
+  padding-top: 0;
```
Comment (oleterno-equinor) [inline src/components/page-create-application/style.css:18]: Either set `padding: var(--eds_spacing_medium)` or `padding-bottom: var(--eds_spacing_medium)`

File: src/components/page-environment/style.css [hunk 4]
```diff
@@ -1,18 +1,21 @@
...
+  overflow-y: auto;
+}
+@media (min-width: 20rem) {
+  .environment-alerting-content {
+    width: 18rem;
+  }
+}
+@media (min-width: 50rem) {
+  .environment-alerting-content {
+    width: 25rem;
```
Comment (oleterno-equinor) [inline src/components/page-environment/style.css:19]: I would suggest increasing the width of the scrim a little bit on larger screens. 
```suggestion
    width: 28rem;
```

File: src/components/page-pipeline-jobs/style.css [hunk 5]
```diff
@@ -7,3 +7,19 @@
...
+  overflow-y: auto;
+}
+@media (min-width: 20rem) {
+  .application-alerting-content {
+    width: 18rem;
+  }
+}
+@media (min-width: 50rem) {
+  .application-alerting-content {
+    width: 25rem;
```
Comment (oleterno-equinor) [inline src/components/page-pipeline-jobs/style.css:23]: I would suggest increasing the width of the scrim a little bit on larger screens.
```suggestion
    width: 28rem;
```

File: src/style/objects.accordion.css [hunk 6]
```diff
@@ -1,22 +1,23 @@
...
-.accordion > div > h4, .accordion > div > p {
-  color: inherit;
-  flex: 1;
-}
-.accordion > div > svg {
-  height: 24px;
-  width: 24px;
-  fill: var(--eds_interactive_primary__resting);
+  border: 1px solid var(--eds_ui_background__medium);
+  border-radius: 2px;
```
Comment (oleterno-equinor) [inline src/style/objects.accordion.css:4]: Style border according to wireframes.
```suggestion
  border: 2px solid var(--eds_text_static_icons_dark__secondary);
  border-radius: 4px;
```

File: src/components/data-chart/style.css [hunk 7]
```diff
@@ -27,51 +27,45 @@
   user-select: none;
 }
 
-.scrim-chart .dialog-container {
-  max-width: 50rem;
-  width: 80vw;
+.chart-container {
+  width: 50vw;
```
[the remark this answers is not in the document]
Comment (Pespiri) [reply src/components/data-chart/style.css]: fixed

File: src/components/data-chart/style.css [hunk 8]
```diff
@@ -27,51 +27,45 @@
...
+  padding: var(--eds_spacing_small) var(--eds_spacing_medium);
+  overflow-x: hidden;
 }
-.scrim-chart .dialog-container .dialog-content {
-  display: grid;
-  gap: var(--eds_spacing_medium);
+.chart-container > *:not(:last-child) {
+  padding-bottom: var(--eds_spacing_medium);
 }
+
```
[the remark this answers is not in the document]
Comment (Pespiri) [reply src/components/data-chart/style.css]: removed styling and added "grid grid--gap-medium" to class

File: src/components/page-pipeline-jobs/style.css (hunk 5 above)
Comment (Pespiri) [reply src/components/page-pipeline-jobs/style.css]: fixed

File: src/components/page-environment/style.css (hunk 4 above)
Comment (Pespiri) [reply src/components/page-environment/style.css]: fixed

File: src/components/page-configuration/style.css (hunk 2 above)
Comment (Pespiri) [reply src/components/page-configuration/style.css:12]: fixed

File: src/style/objects.accordion.css (hunk 6 above)
Comment (Pespiri) [reply src/style/objects.accordion.css]: fixed

File: src/components/page-create-application/style.css (hunk 3 above)
Comment (Pespiri) [reply src/components/page-create-application/style.css]: fixed

**Review by oleterno-equinor — APPROVED** [review]

**Pull request merged**